spec/exporters/odt/cv_spec.rb

```diff
@@ -27,5 +27,10 @@
       expect(nationalities).to eq('Australien')
     end
 
+    it 'notes formatting' do
```

Comment:
Always formulate the test name by creating a sentence using the `it` keyword. 
In this case, this would probably result in `it 'formats competence notes'`